botocore/data/s3/2006-03-01/service-2.sdk-extras.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1.0,
+  "merge": {
+    "metadata":{
+      "protocol":"s3",
+      "protocols":["s3"]
```

Comment:
This piece seems a little concerning. We can probably assume this will never expand for a service like s3, but how do we know if it does? We may want some form of validation on the base model via a test.